include/glow/Partitioner/Partitioner.h

```diff
@@ -157,6 +157,10 @@ class Partitioner {
   /// devices.
   Partitioner(Module *parent, const std::vector<DeviceInfo> &devices);
 
+  /// Decompose a function based on a config.
+  llvm::Error PartitionFromConfig(const std::string &funcName,
```

Comment:
Please pass StringRef here. std::string forces you to malloc twice, while string-ref gives you the option to pass a regular c-string.